mfthomps/CyberCIEGE-port-to-Unity3d
# Issue/53 asset factory

Closes #53 

Similar to #52, this adds a factory instance which creates AssetBehavior GameObjects via a prefab. This removes the Asset game object in the scene and the need to FindObject().

To verify:
* run the Instruction Scenario
* Notice a few "Asset..." GameObjects in the scene.
* I guess these Assets are used in a UI we haven't implemented yet.
